tools/clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -1137,6 +1137,11 @@ llvm::MDNode *CodeGenFunction::getRangeForLoadFromType(QualType Ty) {
   return MDHelper.createRange(Min, End);
 }
 
+bool ShouldEmitRangeMD(llvm::Value *Value, QualType Ty) {
```

Comment:
Shouldn't this be static?  Plus the name is generic, but seems to only apply to bool?  Is that correct?

Reply:
Good catch!

Reply:
```suggestion
static bool ShouldEmitRangeMD(llvm::Value *Value, QualType Ty) {
```